schema/diasource.avsc

```diff
@@ -0,0 +1,176 @@
+			{"name": "apMeanSb02Sigma", "aliases": ["base_CircularApertureFlux_4_5_fluxSigma"], "type": ["float", "null"]},
+			{"name": "apMeanSb03Sigma", "aliases": ["base_CircularApertureFlux_6_0_fluxSigma"], "type": ["float", "null"]},
+			{"name": "apMeanSb04Sigma", "aliases": ["base_CircularApertureFlux_9_0_fluxSigma"], "type": ["float", "null"]},
+			{"name": "apMeanSb05Sigma", "aliases": ["base_CircularApertureFlux_12_0_fluxSigma"], "type": ["float", "null"]},
+			{"name": "apMeanSb06Sigma", "aliases": ["base_CircularApertureFlux_17_0_fluxSigma"], "type": ["float", "null"]},
+			{"name": "apMeanSb07Sigma", "aliases": ["base_CircularApertureFlux_25_0_fluxSigma"], "type": ["float", "null"]},
+			{"name": "apMeanSb08Sigma", "aliases": ["base_CircularApertureFlux_35_0_fluxSigma"], "type": ["float", "null"]},
+			{"name": "apMeanSb09Sigma", "aliases": ["base_CircularApertureFlux_50_0_fluxSigma"], "type": ["float", "null"]},
+			{"name": "apMeanSb10Sigma", "aliases": ["base_CircularApertureFlux_70_0_fluxSigma"], "type": ["float", "null"]}
+				]}, "null"]},
+		{"name": "snr", "type": "float"},
+		{"name": "psFlux", "type": "float"},
+		{"name": "psRadec", "type": [{"type": "record",
+			"name": "psRadec", "fields": [{"name": "psRa", "type": "double"}, {"name": "psDec", "type": "double"}]},
+			"null"]},
+		{"name": "psCov", "type": [{"type": "record",
+			"name": "psCov", "fields": [
+				{"name": "psCov01", "type": "float"},
+				{"name": "psCov02", "type": "float"},
+				{"name": "psCov03", "type": "float"},
+				{"name": "psCov04", "type": "float"},
+				{"name": "psCov05", "type": "float"},
+				{"name": "psCov06", "type": "float"}
+					]},
+			"null"]},
+		{"name": "psFluxSigma", "type": ["float", "null"]},
+		{"name": "psLnL", "type": ["float", "null"]},
+		{"name": "psChi2", "type": ["float", "null"]},
+		{"name": "psN", "type": ["int", "null"]},
+		{"name": "trailFlux", "type": ["float", "null"]},
+		{"name": "trailRadec", "type": [{"type": "record",
+			"name": "trailRadec", "fields": [{"name": "trailRa", "type": "double"}, {"name": "trailDec", "type": "double"}]},
+			"null"]},
+		{"name": "trailCov", "type": [{"type": "record",
+			"name": "trailCov", "fields": [
+				{"name": "trailCov01", "type": "float"},
+				{"name": "trailCov02", "type": "float"},
+				{"name": "trailCov03", "type": "float"},
+				{"name": "trailCov04", "type": "float"},
+				{"name": "trailCov05", "type": "float"},
+				{"name": "trailCov06", "type": "float"},
+				{"name": "trailCov07", "type": "float"},
+				{"name": "trailCov08", "type": "float"},
+				{"name": "trailCov09", "type": "float"},
+				{"name": "trailCov10", "type": "float"},
+				{"name": "trailCov11", "type": "float"},
+				{"name": "trailCov12", "type": "float"},
+				{"name": "trailCov13", "type": "float"},
+				{"name": "trailCov14", "type": "float"},
+				{"name": "trailCov15", "type": "float"}
+					]},
+			"null"]},
+		{"name": "trailFluxSigma", "type": ["float", "null"]},
```

Comment:
I think that these parameters (`trailFluxSigma`, `trailLengthSigma`, etc.) are included in `trailCov`.  There are 5 `trail` parameters: `Flux`, `Ra`, `Dec`, `Length`, and `Angle`.  The covariance matrix for 5 parameters would have 15 unique elements, making up the 15 `float`s in `trailCov`.